From f520116e9a532d057a389ad78729da4b1845bd32 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Wed, 24 Aug 2016 20:27:52 +0000
Subject: [PATCH] MORAY-354 Moray should ship client tools Reviewed by: Dave
 Pacheco <dap@joyent.com> Approved by: Dave Pacheco <dap@joyent.com>

---
 package.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package.json b/package.json
index 85cd0e2..5dd5033 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
         "lru-cache": "2.5.0",
         "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
         "manatee": "git+https://github.com/joyent/manatee.git#8615736",
+        "moray": "git+https://github.com/joyent/node-moray.git#fd5781b",
         "microtime": "0.5.1",
         "once": "1.3.0",
         "panic": "0.2.1",
-- 
2.21.0

